operator/controllers/external_scaler.go

```diff
@@ -47,6 +48,7 @@ func createExternalScaler(
 			},
 		},
 		k8s.Labels(appInfo.ExternalScalerDeploymentName()),
+		env.GetOr("SCALER_PULL_POLICY", "Always"),
```

Comment:
@khaosdoctor can we pass this and the other `*_PULL_POLICY` values down from the controller's `main.go`. It's more work, but it's also helpful to have most/all of the config centralized